scripts/generator/templates/template-release-notes.md

```diff
@@ -1,187 +1,132 @@
-
-### Tools
-- Vaadin Designer **(Pro)** ([Release notes](https://github.com/vaadin/designer/blob/master/RELEASE-NOTES.md))
-- Vaadin TestBench **(Pro)** ([{{vaadin.vaadin-testbench.javaVersion}}](https://github.com/vaadin/testbench/releases/tag/{{vaadin.vaadin-testbench.javaVersion}}))
-
-# Getting Started with Vaadin
-## App starters
-The best way to get started with Vaadin is to go to [https://start.vaadin.com](https://start.vaadin.com) and configure your new application by setting up your views, entities, styles, and the technology stack you’re interested in. 
-
-### Note
-vaadin 23 starters are not available just yet in vaadin.com. You can use vaadin 22 starter and manually change Vaadin version (see instructions below).
-
-## Maven Archetypes
-Maven is the de-facto build tool for Java web applications. Major IDEs also support Maven out of the box and most often you'll be using Maven via your favorite IDE.
-There are currently two Maven archetypes available, the `vaadin-archetype-application` which corresponds to the project base for Flow and the corresponding `vaadin-archetype-spring-application` if you prefer use Flow with [Spring](https://spring.io/).
-The version of the archetype should match the platform version. After you have Maven installed, you can quickly create and run a Vaadin app with the following command:
-
-```
-mvn -B archetype:generate \
-                -DarchetypeGroupId=com.vaadin \
-                -DarchetypeArtifactId=vaadin-archetype-application \
-                -DarchetypeVersion={{platform}}\
-                -DgroupId=org.test \
-                -DartifactId=vaadin-app \
-                -Dversion=1.0-SNAPSHOT \
-                && cd vaadin-app \
-                && mvn package jetty:run
-```
-
-```
-mvn -B archetype:generate \
-                -DarchetypeGroupId=com.vaadin \
-                -DarchetypeArtifactId=vaadin-archetype-spring-application \
-                -DarchetypeVersion={{platform}}\
-                -DgroupId=org.test \
-                -DartifactId=vaadin-app \
-                -Dversion=1.0-SNAPSHOT \
-                && cd vaadin-app \
-                && mvn
-```
-
-## Manually changing Vaadin version for Java projects
-
-Add the following dependency to dependencyManagement in pom.xml.
-```
-<dependency>
-    <groupId>com.vaadin</groupId>
-    <artifactId>vaadin-bom</artifactId>
-    <version>{{platform}}</version>
-    <type>pom</type>
-    <scope>import</scope>
-</dependency>
-```
-
-Read more about upgrading to vaadin 23 from [vaadin.com](https://vaadin.com/docs/latest/guide/upgrading).
-
-
-# Supported Technologies
-## Operating Systems
-Development is supported with the following operating systems:
-- Windows
-- Linux
-- macOS
+Vaadin 23 is supported for one month after Vaadin 24 has been released ([release model](https://vaadin.com/roadmap)).
```

Comment:
Vaadin 23 is a LTS, so we will need to update the statement here.

Reply:
Ah, right. I did remove the mention about 14 LTS, but didn't pay attention how this sentence was formulated.

Reply:
no worries.. that is not your fault.. the original content is wrong too.

Reply:
```suggestion
<!-- New LTS:

Vaadin 23 is an LTS release, and is supported for 5 years, with extended support options available ([release model](https://vaadin.com/roadmap)).

-->

<!-- Non-LTS:

Vaadin 24 is supported for one month after Vaadin 25 has been released ([release model](https://vaadin.com/roadmap)).

-->
```

Reply:
I added two options to choose from. Need to uncomment the one to use.